index.bs

```diff
@@ -76,6 +76,7 @@ spec: web-otp; urlPrefix: https://wicg.github.io/web-otp
 <pre class="link-defaults">
 spec:html; type:dfn; for:html-origin-def; text:origin
 spec:html; type:dfn; for:environment settings object; text:global object
+spec:html; type:dfn; for:navigable; text:parent
```

Comment:
If you use `[=navigable/parent=]` instead of `[=parent=]`, you can avoid this line.